### DISCUSSION TITLE
[Feature Request] Stepper Animation Docs

### DISCUSSION THREAD

**Discussion by kyjus25** [primefaces/community#1489] [PrimeNG] [answered]
I'm sure you assumed this was bound to be asked 😀 

Would love a docs section in the new p-stepper component to demonstrate how to perform fade/slide animations.

Attached is the current implementation I've made using p-steps and would love to be able to migrate, but was curious how this slide animation could be achieved with the new component. 

The content slides to the left when going forward, and slides to the right when going backward.

https://github.com/primefaces/community/assets/6710794/80cbf03c-f5ac-45c7-b97f-754010f2a77f

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, it's a good enhancement, we'll implement this in the feature.
